scripts/test_oppijanumerorekisteri.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Oliko tämä tarkoitus poistaa tässä PR:ssä? Ei oikein liity asiaan mutta toki saa siivota.

Reply:
Poistin samassa syssyssä, kun näin sen ja tiesin ettei tehdä sillä mitään